rel/files/mongooseim

```diff
@@ -118,7 +118,7 @@ case "$1" in
         if [ "$RES2" = "pong" ]; then
             RES3=`$ADD_TO_CLUSTER_CMD`
             if [ "$RES3" = "ok" ]; then
-                echo "Node added to cluster. Run mongooseimctl live or mongooseimclt start."
+                echo "Node added to cluster. Run mongooseimctl live or mongooseimctl start."
```

Comment:
You win a golden magnifying glass !

![lupa](https://cloud.githubusercontent.com/assets/6969609/13844645/418342a8-ec3b-11e5-9acf-e6d3293af7b1.jpg)